### PR TITLE
Shreyalocal

### PR DESCRIPTION
**Pull request by shreyabhattacharya24**
Added details and image of self

### CODE REVIEW

File: members/shreya/data.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "Shreya2406",
```
Comment (strider99) [inline members/shreya/data.json:2]: this should be shreya

File: members/shreya/data.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "first_name": "Shreya",
+  "last_name": "Bhattacharya",
+  "yoe": 8,
+  "company": "Adobe Pvt. Ltd.",
+  "designation": "Lead Software Engineer",
+  "img": "URL to profile pic",
```
Comment (strider99) [inline members/shreya/data.json:8]: "img": "./img.png",

**Comment by shreyabhattacharya24** [comment]
@strider99 -Please review and merge latest changes.

**Comment by ankushdharkar** [comment]
@shreyabhattacharya24 Also, ID missing from `mappings` file

File: ids/mapping.json [hunk 3]
```diff
@@ -10,7 +10,6 @@
   "shubham": "Shubham-1994",
   "sumitdhanania": "sumitd94",
   "shihab": "shihabus",
-  "shreya":"shreya2406",
```
Comment (strider99) [inline ids/mapping.json]: here it should be `"shreya":"shreyabhattacharya24",`

**Comment by swarajpure** [comment]
@shreyabhattacharya24 Are you going to update this PR?

**Comment by swarajpure** [comment]
PR stale for many days. Author did not reply/made requested changes. Hence, closing.

**Pull request closed without merging**